airflow/dag_processing/manager.py

```diff
@@ -526,6 +526,12 @@ def deactivate_stale_dags(
         dags_parsed = session.execute(query)
 
         for dag in dags_parsed:
+            # When the DAG processor runs as part of the scheduler, and the user changes the DAG home folder,
+            # DAGs from the previous DAG home will be marked as stale. Note that this change has no impact
```

Comment:
```suggestion
            # When the DAG processor runs as part of the scheduler, and the user changes the DAGs folder,
            # DAGs from the previous DAGs folder will be marked as stale. Note that this change has no impact
```

nit: We don't really have a "DAG home folder", we have an Airflow home folder and a DAG folder - but those are different things ultimately.